config.go

```diff
@@ -65,6 +65,9 @@ type Configuration struct {
 	Provides      bool   `json:"provides"`
 	PGPFetch      bool   `json:"pgpfetch"`
 	ShowDiffs     bool   `json:"showdifs"`
```

Comment:
This item is now unused, can it be removed?

Reply:
I removed it from the defaultSettings function and all the documentation but forgot to remove it from the actual struct.

If you want to remove it yourself feel free.